index.d.ts

```diff
@@ -1,3 +1,14 @@
+/**
+ * If given to parse, this callback will be invoked per each found emoji.
+ *
+ * If this callback returns a falsy value instead of a valid `src` to use for the image, nothing will actually change for that specific emoji.
+ *
+ * @param icon the lower case HEX code point i.e. "1f4a9"
+ * @param options all info for this parsing operation
+ * @param variant the optional \uFE0F ("as image") variant, in case this info is anyhow meaningful. By default this is ignored.
+ */
+declare type ParseCallback = (icon: string, options: object, variant: string) => string | false;
```

Comment:
shouldn't the type of options stay `TwemojiOptions` instead of `object` ?

Reply:
Why is this `object`?